skyvern/forge/sdk/routes/agent_protocol.py

```diff
@@ -292,16 +293,28 @@ async def cancel_task(
             status_code=status.HTTP_404_NOT_FOUND,
             detail=f"Task not found {task_id}",
         )
-    await app.agent.update_task(task_obj, status=TaskStatus.canceled)
+    task = await app.agent.update_task(task_obj, status=TaskStatus.canceled)
```

Comment:
shall we just only send the webhook when task/workflow cancellation is finished? any specific reason why we must send the webhook in the API call?

Reply:
users can also cancel tasks that are stuck. our worker won't really send a webhook if we don't send a webhook to them in the API endpoint.

Reply:
i think it's okay that we send two cancel task webhook.